pkg/validators/validators.go

```diff
@@ -47,6 +49,25 @@ func handleClientError(e error) error {
 	return e
 }
 
+// TestModuleNotUsed validates that all modules referenced in the "use" field
+// of the blueprint are actually used, i.e. the outputs and settings are
+// connected.
+func TestModuleNotUsed(unusedModules map[string][]string) error {
+	foundUnused := false
+	for mod, unusedMods := range unusedModules {
+		foundUnused = true
+		for _, unusedMod := range unusedMods {
+			log.Printf(unusedModuleMsg, mod, unusedMod)
+		}
+	}
+
+	if foundUnused {
+		return fmt.Errorf("one or more used variables could not have their settings and outputs linked")
```

Comment:
"used variables" should probably read "used modules"

Reply:
Fixed